cmd/sample-file-server/main.go

```diff
@@ -2,8 +2,10 @@ package main
 
 import (
 	"fmt"
+	"image"
+	"image/color"
+	"image/png"
 	"log"
-	"math/rand"
```

Comment:
Slight regression here in that all files are now identical. Probably not too hard to fix if we care a lot about that.